Source/WebCore/dom/Element.cpp

```diff
@@ -2099,28 +2099,11 @@ void Element::attributeChanged(const QualifiedName& name, const AtomString& oldV
         } else if (name == HTMLNames::langAttr || name.matches(XMLNames::langAttr)) {
             if (document().documentElement() == this)
                 document().setDocumentElementLanguage(newValue);
+            else if (isConnected() && (!UNLIKELY(hasRareData()) || elementRareData()->effectiveLang().isNull()) && langFromAttribute() == document().documentElementLanguage())
```

Comment:
I don't think we want to have this isConnected check here.
We probably want to keep track of every element that has the matching lang attribute to be in the set.